docs/DeveloperGuide.adoc

```diff
@@ -775,12 +1230,48 @@ Use case resumes at step 2.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Should work on both 32 bit and 64 bit environment.
+.  Should have features to protect data of persons
```

Comment:
You try to be a bit more specific here. What kinds of features are you thinking of the protect the user data?